pkg/antctl/antctl.go

```diff
@@ -431,6 +454,60 @@ var CommandList = &commandList{
 			},
 			transformedResponse: reflect.TypeOf(controllernetworkpolicy.EndpointQueryResponse{}),
 		},
+		{
+			use:   "flowrecords",
+			short: "Print the matched flow records in the flow aggregator",
+			long:  "Print the matched flow records in the flow aggregator. It supports the 5-tuple flow key or a subset of 5-tuples as a filter.",
```

Comment:
s/matched/matching

s/a subset of 5-tuples/a subset of the 5-tuple

Reply:
done